lib/new_relic/agent/configuration/default_source.rb

```diff
@@ -1956,7 +1956,7 @@ def self.enforce_fallback(allowed_values: nil, fallback: nil)
             If `true`, the agent will operate in a streamlined mode suitable for use with short-lived serverless
               functions. NOTE: Only AWS Lambda functions are supported currently and this option is not intended for use
               without
-              [New Relic's Ruby Lambda layer](https://docs.newrelic.com/docs/serverless-function-monitoring/aws-lambda-monitoring/get-started/monitoring-aws-lambda-serverless-monitoring/)
+              [New Relic's Ruby Lambda layer based instrumentation](https://docs.newrelic.com/docs/serverless-function-monitoring/aws-lambda-monitoring/instrument-lambda-function/instrument-your-own/)
```

Comment:
```suggestion
              [New Relic's Ruby Lambda layer instrumentation](https://docs.newrelic.com/docs/serverless-function-monitoring/aws-lambda-monitoring/instrument-lambda-function/instrument-your-own/)
```
Maybe?

Reply:
In this case it's a layer that's delivering the instrumentation, and not itself being instrumented. So I tried to reflect that disambiguation with "layer based".